tortoise/recover.go

```diff
@@ -20,7 +19,7 @@ import (
 // Recover tortoise state from database.
 func Recover(
 	ctx context.Context,
-	db *datastore.CachedDB,
+	db *sql.Database,
```

Comment:
How about using `sql.Executor` where possible?
```suggestion
	db sql.Executor
```